### PR TITLE
HardFork announcements

### PR DESCRIPTION
**Pull request by renzobarrionuevo**

### CODE REVIEW

File: HardFork announcements.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**_Plan for the next update of the Bitcoin Cash network_**
+
+We publish this information so that Bitprim node users and other interested parties are aware that BitcoinCash has a protocol update on November 15. All node operators must be updated to remain compatible with the Bitcoincash network. The following timeline denotes the processes that will take place before the stipulated date for the Hard Fork.
+
+### August 15th: Code end date
+
+Final date for the presentation of the package of changes that will be included in the following hardfork. After this date, the test stage will begin.
+
+Any other change of protocol that wants to be included after August 15 will be pending to be reviewed for the Hard Fork of May 2019.
```
Comment (dario-ramos) [inline HardFork announcements.md]: Any other protocol change proposed after August 15*

File: HardFork announcements.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### October 15th: Testnet deadline & official release
+
+Once the tests have been completed and everything works as expected, Bitprim will officially launch its new version of the node. The node operators will have 30 days to apply the update before the hardfork.
```
Comment (dario-ramos) [inline HardFork announcements.md]: hard fork

File: HardFork announcements.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### November 15: Hard Fork
+
+Any node operator that has not updated for the date will be running outside of the Bitcoin Cash protocol.
```
Comment (dario-ramos) [inline HardFork announcements.md]: that has not updated at this date

File: HardFork announcements.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+follow as on https://twitter.com/bitprim
```
Comment (dario-ramos) [inline HardFork announcements.md]: follow us

File: HardFork announcements.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Timeline for next upgrade
+
+**_Plan for the next update of the Bitcoin Cash network_**
+
+We publish this information so that Bitprim node users and other interested parties are aware that BitcoinCash has a protocol update on November 15. All node operators must be updated to remain compatible with the Bitcoincash network. The following timeline denotes the processes that will take place before the stipulated date for the Hard Fork.
```
Comment (mdallago) [inline HardFork announcements.md]: BitcoinCash or Bitcoint Cash network....

File: HardFork announcements.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+### August 15th: Code end date
+
+Final date for the presentation of the package of changes that will be included in the following hardfork. After this date, the test stage will begin.
```
Comment (mdallago) [inline HardFork announcements.md]: hardfork or Hard Fork ?

Comment (dario-ramos) [reply HardFork announcements.md]: Two words: Adjective (hard) + Noun (fork), although some may consider it a portmanteau. Choose one and use it consistently.

File: HardFork announcements.md (hunk 5 above)
Comment (dario-ramos) [reply HardFork announcements.md]: Choose one and use it consistently.

File: HardFork announcements.md (hunk 1 above)
Comment (renzobarrionuevo) [reply HardFork announcements.md]: Corrected

File: HardFork announcements.md (hunk 2 above)
Comment (renzobarrionuevo) [reply HardFork announcements.md]: Corrected

**Pull request merged**